ci-operator/step-registry/assisted/vsphere/setup/template/assisted-vsphere-setup-template-commands.sh

```diff
@@ -23,17 +23,19 @@ declare vsphere_portgroup
 source "${SHARED_DIR}/vsphere_context.sh"
 SSH_PUBLIC_KEY="$(cat /var/run/vault/assisted-ci-vault/ssh_public_key)"
 
+echo "Creating packer build path"
 mkdir -p build/packer
 cp -r packer_files/vsphere_centos_template/* build/packer/
 cd build/packer/
 
-# Get Centos Stream 8 latest version checksum
-STREAM8_ISO="CentOS-Stream-8-x86_64-latest-boot.iso"
-ISO_CHECKSUM=$(curl -s http://rep-centos-il.upress.io/8-stream/isos/x86_64/CHECKSUM | grep "${STREAM8_ISO}" | awk -F' = ' '{print $2}' | tr -d '\n')
+echo "Get Centos Stream latest version checksum"
+CENTOS_ISO="$(cat /var/run/vault/assisted-ci-vault/centos_iso_image_name)"
+CENTOS_ISO_BASE_PATH="$(cat /var/run/vault/assisted-ci-vault/centos_iso_image_url)"
```

Comment:
@eliorerz since this is in a secret in vault can you echo the url you are using?

The negative of the curl & pipes is when this fails, and I am testing it right now and it does, you don't know why it is failing.

Reply:
testing here...
https://github.com/openshift/release/pull/52907

but of course doesn't include these changes

Reply:
The values are as followed:
CENTOS_ISO_BASE_PATH=https://mirror.stream.centos.org/9-stream/BaseOS/x86_64/iso/
CENTOS_ISO=CentOS-Stream-9-latest-x86_64-boot.iso